repos/spack_repo/builtin/packages/whisper_cpp/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+
+from spack.package import *
+
+
+class WhisperCpp(CMakePackage, CudaPackage):
+    """High-performance inference of OpenAI's Whisper automatic speech recognition (ASR) model"""
+
+    homepage = "https://github.com/ggml-org/whisper.cpp"
+    url = "https://github.com/ggml-org/whisper.cpp/archive/refs/tags/v1.8.2.tar.gz"
+    git = "https://github.com/ggml-org/whisper.cpp"
+
+    license("MIT", checked_by="aweits")
+
+    variant("sdl2", default=False, description="sdl2/streaming support")
+
+    depends_on("cuda")
+    depends_on("curl")
+    depends_on("sdl2", when="+sdl2")
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
```

Comment:
```suggestion
    depends_on("c", type="build")
    depends_on("cxx", type="build")
    
    depends_on("cuda")
    depends_on("curl")
    
    depends_on("sdl2", when="+sdl2")
```

We should break out the language, required, and variant dependencies following the common format in other packages.